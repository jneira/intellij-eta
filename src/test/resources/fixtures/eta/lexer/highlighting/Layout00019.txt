ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00019')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('f')
WHITE_SPACE (' ')
ITvarid ('x')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE ('  ')
ITvarid ('maybeEP')
WHITE_SPACE (' ')
IToparen ('(')
ITvarsym ('\')
ITvarid ('ids')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n                  ')
ITvarid ('layoutList')
WHITE_SPACE (' ')
ITvarid ('pts'')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('sepInstFunBinds')
WHITE_SPACE (' ')
ITvarid ('ids')
WHITE_SPACE ('\n                  ')
ITcparen (')')
WHITE_SPACE (' ')
ITvarid ('mids')
WHITE_SPACE ('\n')
