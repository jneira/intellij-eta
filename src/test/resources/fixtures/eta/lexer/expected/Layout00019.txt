ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00019')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
ITvarid ('f')
WHITE_SPACE (' ')
ITvarid ('x')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE ('  ')
ITvarid ('maybeEP')
WHITE_SPACE (' ')
IToparen ('(')
ITlam ('\')
ITvarid ('ids')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n                  ')
ITvocurly ('')
ITvarid ('layoutList')
WHITE_SPACE (' ')
ITvarid ('pts'')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('sepInstFunBinds')
WHITE_SPACE (' ')
ITvarid ('ids')
WHITE_SPACE ('\n                  ')
ITsemi ('')
ITcparen (')')
WHITE_SPACE (' ')
ITvarid ('mids')
WHITE_SPACE ('\n')
ITvccurly ('')
ITsemi ('')